backend/census_historical_migration/make_submission.py

```diff
@@ -0,0 +1,44 @@
+from .workbooklib.end_to_end_core import run_end_to_end
+from .models import ELECAUDITHEADER
+
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+def load_historic_data(audit_year, dbkey):
+    """Create a submission for the given audit year and dbkey"""
+    result = {"success": [], "errors": []}
```

Comment:
It does not seem that `errors` is being used in create_or_get_user and run_end_to_end.

Reply:
It is probably okay to have a light weight error check here. Later in the process we'll craft a more refined mechanism to catch errors and record in the database

Reply:
There's a tough design decision that always plagues this kind of software: throw exceptions, or return errors?

In Go, the practice is to return `error` objects. In Java, it's exceptions. Python... could go either way.

It is possible that an exception-based mechanism will allow y'all to short-circuit a migration that is going to be a "failure," but it might also be that we want to attempt a full migration of every record, and then record *all* the ways that record failed, so that we have more data for the next round. 

Something to noodle on.